### ISSUE TITLE
Perceptual Loss

### ISSUE THREAD

**Issue by vishnusanjayrs** [danielism97/ST-MFNet#14]
Thanks for sharing your amazing work. 

In the paper you mentioned that  "  adversarial loss for the generator is combined with the Laplacian pyramid loss to
form the perceptual loss for ST-MFNet fine-tuning" can you please let us know the value for lambda used and after how many epochs of training with just Laplacian pyramid loss this perceptual loss term was introduced. 

Thanks

**Comment by danier97** [comment]
Hi,

Thank you for your interest in the work!

The model was trained with the Laplacian loss for around 70 epochs. Then fine-tuning was performed for a further 10 epochs with $\lambda$=100.

Thanks.

**Comment by vishnusanjayrs** [comment]
Thanks for the quick reply
